packages/estransform/build.js

```diff
@@ -12,4 +12,9 @@ esbuild.build({
     external: [
         '@parcel/source-map',
     ],
+    banner: {
+        js: `import { createRequire as __moduleCreateRequire } from 'module';
+const require = __moduleCreateRequire(import.meta.url);
+`,
+    },
```

Comment:
I used `npm run build` as a way to test this PR and this is how I found out that this `require` is needed in the new output

any other tests I can run? I didn't understand it from `package.json` or docs

I tested on my side and the new dist works